source/appController/AMDSClientAppController.h

```diff
@@ -48,24 +49,26 @@ class AMDSClientAppController : public AMDSAppController
 	void openNetworkSession();
 
 	/// request to establish a connection to a specific hostName and the portNumber
+	void connectToServer(const AMDSServerConfiguration &serverConfiguration);
+	/// request to establish a connection to a specific hostName and the portNumber
 	void connectToServer(const QString &hostName, quint16 portNumber);
 	/// request disconnection with a given server identifier
 	void disconnectWithServer(const QString &serverIdentifier);
 
 	/// request data from server for Statistics
-	void requestClientData(const QString &hostName, quint16 portNumber);
+	bool requestClientData(const QString &hostName, quint16 portNumber);
```

Comment:
Only question I have is if these `requestClientData()` methods should be const or not?